test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -494,7 +495,11 @@ def setUpClass(cls):
         cls.parse_param()
         cls.init_param()
         cls.check_users_exist()
+        _, path = tempfile.mkstemp(prefix="setupclass_output",
+                                   suffix=".json")
+        cls.saved_file = path
         cls.init_servers()
+        cls.server.save_configuration(path, 'w')
```

Comment:
trying to understand the logic, shouldn't you be saving the configuration only when '--use-current-setup' was passed by the user?

Reply:
As per EDD system should revert to original setup after running PTL even when --use-current-setup is not provided in benchpress command

Reply:
Since saving configuration is kind of mandatory now, I think in the `save_configuration()`, you are returning `False` so maybe you should add a check here as well to proceed only if it succeeds.